runtime/lua/vim/lsp.lua

```diff
@@ -585,8 +585,10 @@ do
   text_document_did_change_handler = function(_, bufnr, changedtick,
       firstline, lastline, new_lastline, old_byte_size, old_utf32_size,
       old_utf16_size)
-    local _ = log.debug() and log.debug("on_lines", bufnr, changedtick, firstline,
-    lastline, new_lastline, old_byte_size, old_utf32_size, old_utf16_size, nvim_buf_get_lines(bufnr, firstline, new_lastline, true))
+
+    local formatted_log = string.format("on_lines bufnr: %s, changedtick: %s, firstline: %s, lastline: %s, new_lastline: %s, old_byte_size: %s, old_utf32_size: %s, old_utf16_size: %s",
+      bufnr, changedtick, firstline, lastline, new_lastline, old_byte_size, old_utf32_size, old_utf16_size)
+    local _ = log.debug() and log.debug(formatted_log, nvim_buf_get_lines(bufnr, firstline, new_lastline, true))
```

Comment:
can you put `formatted_log` directly in the log.debug call? There's no reason to save the variable if we're not going to log it.

Reply:
I've fixed it :)